skada/metrics.py

```diff
@@ -413,3 +422,105 @@ def _score(self, estimator, X, y, sample_domain=None, **kwargs):
         var_w = np.var(weights, ddof=1)
         eta = -cov / var_w
         return self._sign * (np.mean(weighted_error) + eta * np.mean(weights) - eta)
+
+
+class CircularValidation(_BaseDomainAwareScorer):
+    """Score based on a circular validation strategy.
+
+    See [1]_ for details.
```

Comment:
Bad number : use [21] as in teh redame

Reply:
Also small sentence to explain that this estimator retrain an estimator for each call this should be in the descriptin